src/clj/game/core/io.clj

```diff
@@ -170,6 +170,9 @@
                                             (rez state side c {:ignore-cost :all-costs :force true}))))}}}
     {:title "/rez-all command"} nil))
 
+(defn command-roll [state side value]
+  (system-msg state side (str "rolls a " value " sided die and rolls a " (+ (rand-int (- value 1)) 1))))
```

Comment:
Clojure-fu: instead of `(+ ... 1)`, use `(inc ...)`. :) Likewise, `(dec ...)` instead of `(- ... 1)`.

Reply:
Plus, I now realise my logic was wrong anyway! Consider my wrists slapped for not testing thoroughly enough! 😄